app/controllers/membership_applications_controller.rb

```diff
@@ -45,11 +45,7 @@ def update
 
       new_upload_file params['uploaded_file'] if params['uploaded_file']
 
-      if changed_to_accepted?(params['membership_application'])
-        accept_application
-        redirect_to edit_membership_application_url(@membership_application)
-        return
-      end
+      @membership_application.applicant_updated_info! if (current_user.id == @membership_application.user.id)
```

Comment:
We don't do this yet.  Sounds like a user story & feature that needs to be written up. :-)